_transactions.md

```diff
@@ -117,7 +117,7 @@ The first step is to prepare the transaction by specifying:
 - The _currency_ to denominate the transaction by.
 - The _amount_ of value to send in the denominated currency.
 - The _origin_ of the transaction, which can be an account id in the case of a _deposit_.
-- The _destination_ of the transaction, which can be in the form of a bitcoin address, an email address, an account id or an application id.
+- The _destination_ of the transaction, which can be in the form of a bitcoin address, an email address, an account id, an application id, or a [card id](#card-object).
```

Comment:
We could take this opportunity to update the valid destinations for a transaction. In regards to crypto networks, Bitcoin addresses are not the only ones we support.

Reply:
Great point — especially in light of #208. I'll update.

Reply:
@fmfmartins PR updated.